music_album.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'item'
+
+class MusicAlbum < Item
+  attr_accessor :on_spotify
+
+  def initialize(publish_date, on_spotify)
+    super(id, name, genre, label, publish_date)
```

Comment:
here, please call only included arguments in initialize method in super constructor